### DISCUSSION TITLE
How to convert Data from rrdtool 1.7.0 to 1.8.0?

### DISCUSSION THREAD

**Discussion by kindkains** [oetiker/rrdtool-1.x#1240] [Q&A] [answered]
I've installed a new nagios server with omd and migrating step by step the data from our old nagios to the new one. But simply copying the rrd and xml files from the old server to the new one seems not to work, because I get no graphs in pnp4nagios.

How can I convert the data from the old server in rrdtool version 1.7.0 to the new server with rrdtool version 1.8.0?

**Comment by oetiker** [comment, marked as answer]
The data format between 1.7.0 and 1.8.0 has not changed. Could it be that you are moving between platforms ? In that case use the rrdtool dump and rrdtool restore functions to transfer your file from one platform to the other.
